### PR TITLE
Add ImgBot Swag

### LINKED ISSUE #490: ImgBot Stickers
## Wild swag opportunity appeared!

- [x] I've checked that this isn't a duplicate issue.

Q. Which company offers it?  
A. : ImgBot

Q. Where have you seen the announcement (URL)?  
A. : It is linked at the bottom of all PRs the bot makes: https://github.com/gemssingaporestudentcouncil/innovationweek2020/pull/3

Q. What kind of swag (clothing, stickers, ...)?  
A. : Stickers

Q. Have you received swag from this?  
A. : Not yet, but seen people on Twitter receive it.

Q. How long did it take?  
A. : Unsure

Q. Does it ship worldwide?  
A. : Yes

- [x] I've checked that the promotion is legit.
- [x] I've checked that it's still available to the public.

### PR DESCRIPTION
**Pull request by sampoder**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

I have added the ImgBot Swag which resolves #490

### CODE REVIEW

**Comment by aslafy-z** [comment]
Can you please order alphabetically? Thanks 😊

**Comment by sampoder** [comment]
ok no problem

**Comment by sampoder** [comment]
Fixed @aslafy-z

**Comment by aslafy-z** [comment]
Perfect! Last thing, please adjust the date to when you opened the issue first, on the 2 of January.

**Comment by sampoder** [comment]
Done @aslafy-z

File: data.json [hunk 1]
```diff
@@ -206,6 +206,15 @@
     "dateAdded": "2019-10-02T21:06:00.000Z",
     "tags": ["clothing", "hacktoberfest", "stickers", "expired"]
   },
+  {
+    "name": "ImgBot",
+    "difficulty": "easy",
+    "description": "Install the bot on your repo and get some awesome laptop stickers both from ImgBot after filling out the form!",
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
    "description": "Install the bot on your repo and get some awesome laptop stickers from ImgBot after filling out the form!",
```

**Comment by sampoder** [comment]
@vikaspotluri123 sorry about that! I fixed the sentence.

**Pull request merged**